docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -4,16 +4,19 @@ description: "Trigger a consolidated data query for the given patient"
 api: "POST /medical/v1/patient/{id}/consolidated/query"
 ---
 
-When executed, this endpoint triggers an asynchronous query to retrieve a Patient's consolidated data from
-the FHIR repository.
+When executed, this endpoint triggers an asynchronous query to retrieve a Patient's consolidated
+data from the FHIR repository.
 
 Once the data is consolidated and ready for consumption, a [Webhook request](/medical-api/more-info/webhooks#patient-consolidated-data)
-is sent to your configured URL containing the Patient's data in FHIR-compliant format, based on the filters used to trigger the query.
+is sent to [your configured URL](/home/api-reference/settings/post-settings) containing the Patient's
+data in FHIR-compliant format, based on the filters used to trigger the query.
 
 You can check the status of the data query by calling [get consolidated data query](/medical-api/api-reference/fhir/consolidated-data-query-get).
 
-This endpoint also provides the ability to render a medical record summary from the FHIR bundle as a PDF or HTML document.
-This will be returned [via a webook](/medical-api/more-info/webhooks#medical-record) as a FHIR bundle with a DocumentReference resource that will contain a url to retrieve the data from.
+This endpoint also provides the ability to render a medical record summary from the FHIR bundle as a
+PDF or HTML document. This will be returned [via a webook](/medical-api/more-info/webhooks#medical-record)
+as a FHIR bundle with a [DocumentReference](/medical-api/fhir/resources/documentreference) resource
+that will contain a URL to retrieve the data from.
```

Comment:
Should we add a sentence denoting that they can also get that file in JSON format?

Reply:
Decided to leave as is, did other updates